Formula/mosquitto.rb

```diff
@@ -1,8 +1,8 @@
 class Mosquitto < Formula
   desc "Message broker implementing the MQTT protocol"
   homepage "https://mosquitto.org/"
-  url "https://mosquitto.org/files/source/mosquitto-1.4.12.tar.gz"
-  sha256 "b12e2353f92cb76882ce7b7bddfea022b461b38cbe6e7fb7c969d0daeb379cf7"
+  url "http://mosquitto.org/files/source/mosquitto-1.4.14.tar.gz"
```

Comment:
It looks like you accidentally switched from `https` to `http`. Could you switch it back? Thanks.